Documentation/ApiOverview/DirectoryStructure/Index.rst

```diff
@@ -21,18 +21,76 @@ almost always find the directory structure as outlined below.
 Also see :ref:`Environment` for further information, especially how to retrieve
 the paths within PHP code.
 
+Files on project level
+======================
+
+On the top most level, the project level, you can find the files
+:file:`composer.json` which contains requirements for the TYPO3 installation
+and the :file:`composer.lock` which contains information about the concretely
+installed versions of each package.
+
 Directories in a typical project
 ================================
 
 .. contents::
    :local:
 
+.. _directory-config:
+
+:file:`config/`
+---------------
+
+Composer installations only. TYPO3 configuration directory. This directory
+contains installation-wide configuration.
+
+.. _directory-config-sites:
+
+:file:`config/sites/`
+~~~~~~~~~~~~~~~~~~~~~
+
+Composer installations only.  The folder :file:`config/sites` contains
+subfolders for each :ref:`site configuration <sitehandling>`.
+
+
+.. _directory-local_packages:
+
+:file:`local_packages/`
+-----------------------
+
+Composer installations only.  Each web site which is run on TYPO3 **should**
+have a sitepackage, anextension with a special purpose containing all
+templates, styles, images, etc. needed for the theme.
+
+It is usually stored locally and then symlinked into the :ref:`directory-vendor`
```

Comment:
That is not true for v11 with composer-installers v3 (the majority) - and as this is merged into v11 it should be `typo3conf/ext`.

Reply:
Will be dealed with in a follow up